doc/releases/v03.00.adoc

```diff
@@ -0,0 +1,56 @@
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.0.0[v3.0.0 - OSI "Cosmic Coulthard"]
+
+== Purpose
+
+This release is a complete rework including the feedback from many OSI users. Thank you!
+We are aligned with the ongoing discussions for the upcoming ISO 23150 standard for hardware sensor outputs in order to bring simulation and the hardware in the car closer together.
+
+https://github.com/OpenSimulationInterface/open-simulation-interface/pulls?page=1&q=is%3Apr+is%3Aclosed+milestone%3Av3.0.0[Content/Changes]
```

Comment:
Added missing header markup in context of PR #533 (9ac1d74d03234f5c351f4691c1accc995a9c33c8).